lib/pages/my_pixels/my_pixels.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/widgets.dart';
+import '../../widgets/scaffold.dart';
+import './widgets/header.dart';
+import './widgets/pixels.dart';
+
+class MyPixels extends StatelessWidget {
+  final bool myself;
+  final bool following;
+  final String photoURL;
+  final String name;
+  final String bio;
+  final int postsCount;
+  final int followersCount;
+  final int followingCount;
+  final List<String> pixels;
+
+  MyPixels(
+      {this.myself = true,
+      this.following = false,
+      @required this.photoURL,
+      @required this.name,
+      @required this.bio,
+      @required this.postsCount,
+      @required this.followingCount,
+      @required this.followersCount,
+      @required this.pixels});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+          child: Column(
+        children: [
```

Comment:
Here on GitHub the line indentation looks like difference. It's not very important, so it is just a simple comment, we can check it after.

Reply:
I'm using "editor.formatOnSave" from vscode, maybe we need a .editorconfig file. I'll check it!

Reply:
This is easily fixed with the `flutter format .` command, I think we should always run this before commiting something, what do you think?

Ideally this could be a pre commit, but I am not aware of any flutter/dart pre commiters, and I would not like the idea of adding npm just to do that.